packages/playground/website/src/components/modal/index.tsx

```diff
@@ -27,15 +27,19 @@ export const defaultStyles: ReactModal.Styles = {
 		background: '#1e2327d0',
 	},
 };
+
 export default function Modal(props: ModalProps) {
+	const closeBtnAriaLabel = 'Close import window';
```

Comment:
Since the modal provides current context, can shorten to just Close.

```suggestion
	const closeBtnAriaLabel = 'Close';
```

Reply:
+1